test/include/seqan3/test/tmp_filename.hpp

```diff
@@ -116,24 +110,24 @@ class tmp_file_name
     explicit tmp_file_name(const char * f_name)
     {
         if (f_name == nullptr)
-            throw fs::filesystem_error("Empty file name!",
-                                        std::make_error_code(std::errc::invalid_argument));
+            throw filesystem::filesystem_error("Empty file name!",
+                  std::make_error_code(std::errc::invalid_argument));
```

Comment:
Hm, the alignment should be at the parenthesis:
```cpp
            throw filesystem::filesystem_error("Empty file name!",
                                               std::make_error_code(std::errc::invalid_argument));
```
But please evaluate first, whether it doesn't fit into a single line (120characters maximum). If it doesn't fit into a single line, you need to put the body of the if-statement into `{}`

Reply:
Tks for the tips, not used to the code standards (yet). Hope it's fine now.